backend/api/sessions.py

```diff
@@ -0,0 +1,19 @@
+from fastapi import Request
+
+sessions = {}
```

Comment:
Would it help avoid future confusion if we named this global something like `session_store` or similar?